core/color.cpp

```diff
@@ -412,11 +412,7 @@ Color Color::named(const String &p_name) {
 	name = name.to_lower();
 
 	const Map<String, Color>::Element *color = _named_colors.find(name);
-	if (color) {
-		return color->value();
-	} else {
-		ERR_FAIL_V_MSG(Color(), "Invalid Color Name: " + p_name + ".");
-	}
+	ERR_FAIL_NULL_V_MSG(color, Color(), "Invalid color name: " + p_name + ".");
```

Comment:
The removed `return color->value();` should be after the `ERR_FAIL`. Otherwise there's no correct return.